pkg/proxy/nftables/proxier.go

```diff
@@ -268,7 +268,7 @@ func NewProxier(ctx context.Context,
 
 	burstSyncs := 2
 	logger.V(2).Info("NFTables sync params", "minSyncPeriod", minSyncPeriod, "syncPeriod", syncPeriod, "burstSyncs", burstSyncs)
-	proxier.syncRunner = async.NewBoundedFrequencyRunner("sync-runner", proxier.syncProxyRules, minSyncPeriod, syncPeriod, burstSyncs)
+	proxier.syncRunner = async.NewBoundedFrequencyRunner("sync-runner", proxier.syncProxyRules, minSyncPeriod, time.Hour, burstSyncs)
```

Comment:
Dan added this comment in the kube-proxy so is clear why we pass time.Hour
```
	// We pass syncPeriod to ipt.Monitor, which will call us only if it needs to.
	// We need to pass *some* maxInterval to NewBoundedFrequencyRunner anyway though.
	// time.Hour is arbitrary.
```

Reply:
can you add the same comment here?

Reply:
sorry! it got lost in my branches

Reply:
You can just squash the "Bump frequency to 1 hour" commit in with the rest of the partial sync commit. It's all part of the same thing really ("don't sync more than we have to")